core/src/server/rpc_module.rs

```diff
@@ -403,7 +426,10 @@ impl Methods {
 
 	/// Execute a callback.
 	async fn inner_call(&self, req: Request<'_>) -> RawRpcResponse {
```

Comment:
Now that the sink is only needed for subscriptions, I wonder whether inner_call could return an enum, so it would only need to construct the subscription channels etc
in the event that the method being called was a subscription, else it could do a simpler thing with other types of method?

This might also make it a little more obvious when the returned sink etc are actually useful?